.env

```diff
@@ -0,0 +1,3 @@
+USER_EMAIL = smtp.gmail.com
+EMAIL_PASSWORD = agroeasy1#$##
```

Comment:
How did you get this here ?.
i think i have .env on git ignore

Reply:
@joshuaobinna  ffs remove the `.env` file from your commit.